src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -148,6 +150,31 @@ def execute(self, args):
 
         strategy.execute()
 
+    def _get_strategy(self, args):
+        strategy = get_strategy(args, args.devices)
+        unused_devices = [device for device in args.devices if not device.used_by_ceph]
+        # only data devices, journals can be reused
+        used_devices = [device.abspath for device in args.devices if device.used_by_ceph]
+        args.filtered_devices = {}
+        if used_devices:
+            for device in used_devices:
+                args.filtered_devices[device] = {"reasons": ["Used by ceph as a data device already"]}
+            if not args.report:
+                mlogger.info("Ignoring devices already used by ceph: %s" % ",".join(used_devices))
```

Comment:
if trying to load json from the output, wouldn't this get in the way? (or is it luck that this is going to stderr while json is being sent to stdout)

Reply:
I think that no logging to the terminal should happen with `--report`, and messages like this should be included in the report

Reply:
That is not being logged to the terminal with ``--report``. See the ``if not args.report`` above, this information is included in ``--report`` in the ``filtered_devices`` key.

Reply:
You do get a report as a user if not passing in `--yes` though.

Reply:
Yeah, I still need to update the "pretty" report to include the filtered_devices.